src/app/components/cohort-grid/cohort-grid.service.js

```diff
@@ -41,36 +41,40 @@ angular.module('transmartBaseUi').factory('CohortGridService', ['$timeout', func
                 columnDefs.push({field: label.name, width: service.WIDTH_PER_COLUMN});
             }
         });
+        columnDefs.push({field: 'cohort-panel', width: service.WIDTH_PER_COLUMN});
         return columnDefs;
     };
 
     /**
      * Format data and labels for the gridview's data
      * @memberof CohortGridService
      * @param subjects
-     * @param headers
+     * @param labels
```

Comment:
looks to me the param is now only one, if this should be or then probably should be 

@param subjects or labels